src/main/java/org/parabot/core/ui/BotUI.java

```diff
@@ -143,7 +146,22 @@ public void actionPerformed(ActionEvent e) {
 
         switch (command) {
             case "Create screenshot":
-                JOptionPane.showMessageDialog(this, "We are still working on this...");
+                try {
+                    Robot robot = new Robot();
+                    Rectangle parabotScreen = new Rectangle((int) getLocation().getX(), (int) getLocation().getY(), getWidth(), getHeight());
+                    BufferedImage image = robot.createScreenCapture(parabotScreen);
+                    String randString = StringUtils.randomString(10);
+                    for (int x = 0; x < Directories.getScreenshotDir().listFiles().length; x++) {
```

Comment:
This is not really the appropriate way to solve an issue like this.

Better way would be like (pseudo code):
```
String randString = StringUtils.randomString(10);
Files[] files = Directories.getScreenshotDir().listFiles();

boolean search = true;
boolean duplicate = false;

while(search == true){
    for (File f : files){
        if (f.getName().equals(randString)){
            duplicate = true;
            break;
        }
    }
    if (duplicate == false){
        search = false;
    } else {
        randString = StringUtils.randomString(10)
    }
}
```